book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/3_0_0_take_a_log.md

```diff
@@ -34,7 +34,7 @@ This will only log the imagery, camera_info, the control commands and a few othe
 
 To log everything that is being published, run the base container on the Duckiebot:
 
-    laptop $ dts duckiebot demo --demo_name make_log_full_docker --duckiebot_name ![DUCKIEBOT_NAME] --package_name duckietown_demos --image duckietown/dt-core:daffy
+    laptop $ dts duckiebot demo --demo_name make_log_full_docker --duckiebot_name ![DUCKIEBOT_NAME] --package_name duckietown_demos --image duckietown/dt-core:daffy-arm32v7
```

Comment:
remote --image

Reply:
I left this there specifically this is for Mac option, not sure if removing will have any affect.

Reply:
removing it will have no effect but it's ugly and is more likely to become out of date this way

Reply:
fixed.